traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -14,14 +14,17 @@ public protocol AvailabilityViewControllerDelegate: class {
 }
 
 open class AvailabilityViewController: UIViewController {
```

Comment:
This VC is too bloated. It is not the VCs responsibility to handle the logic of a calendar display.

Make a new UIView subclass that encapsulates all calendar stuff and use it inside this VC..Keep VCs thin